npm/webpack-batteries-included-preprocessor/index.js

```diff
@@ -146,42 +127,42 @@ const getDefaultWebpackOptions = () => {
     resolve: {
       extensions: ['.js', '.json', '.jsx', '.mjs', '.coffee'],
       fallback: {
```

Comment:
Do we need to set everything to `false` or can we just not provide the `fallback` config?